OpenEdXMobile/src/test/java/org/edx/mobile/util/WhatsNewUtilTest.java

```diff
@@ -0,0 +1,79 @@
+package org.edx.mobile.util;
+
+import android.content.Context;
+import android.support.annotation.DrawableRes;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.edx.mobile.R;
+import org.edx.mobile.test.BaseTest;
+import org.edx.mobile.whatsnew.WhatsNewItemModel;
+import org.edx.mobile.whatsnew.WhatsNewModel;
+import org.junit.Test;
+import org.robolectric.RuntimeEnvironment;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.text.ParseException;
+import java.util.List;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+public class WhatsNewUtilTest extends BaseTest {
+    private final String IMAGE_NAME_REGEX = "screen_\\d+";
```

Comment:
`IMAGE_NAME_REGEX` should be a regex for any valid name like the image can be named `whats_new_due_date`.

Reply:
Our screen name pattern defined is "screen_(number)", currently we are forcing it be followed. If we will change it later on due to any good reason we will update the `IMAGE_NAME_REGEX`  correspondingly.